src/styles/normalize.scss

```diff
@@ -0,0 +1,28 @@
+*,
+*::after,
+*::before,
```

Comment:
Is it really necessary to reset everything on everything? 🤔

Reply:
for border-box maybe